Sources/DynamoDBModel/DynamoDBModelStructures.swift

```diff
@@ -536,7 +536,7 @@ public struct BatchGetItemOutput: Codable, Equatable {
     }
 }
 
-public struct BatchStatementError: Codable, Equatable, Hashable {
```

Comment:
I'm not sure why Haotian added this. https://github.com/amzn/smoke-aws/pull/127
This will pop up as an error somewhere. But this still shouldn't be here.